dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.34/V2_34_1__Update_program_rule_action_with_environments_and_evaluation_time_columns_and_default_values.sql

```diff
@@ -0,0 +1,6 @@
+alter table programruleaction add evaluationtime varchar(50);
+alter table programruleaction add environments jsonb;
```

Comment:
By default we try to write upgrade scripts in a way that they can potentially run multiple times. So in this case you can add a "if not exists", like this:

add column if not exists evaluationtime character varying(50);